routers/api/v1/activitypub/person.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package activitypub
+
+import (
+	"net/http"
+	"net/url"
+	"strings"
+
+	"code.gitea.io/gitea/modules/activitypub"
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/setting"
+	"code.gitea.io/gitea/routers/api/v1/user"
+
+	"github.com/go-fed/activity/streams"
+)
+
+// Person function
+func Person(ctx *context.APIContext) {
+	// swagger:operation GET /activitypub/user/{username} activitypub activitypubPerson
+	// ---
+	// summary: Returns the person
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: username
+	//   in: path
+	//   description: username of the user
+	//   type: string
+	//   required: true
+	// responses:
+	//   "200":
+	//     "$ref": "#/responses/ActivityPub"
+
+	user := user.GetUserByParamsName(ctx, "username")
+	if user == nil {
```

Comment:
```suggestion
	if ctx.ContextUser == nil {
```

Reply:
`ctx.ContextUser` seems to always be `nil`.